pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -79,20 +79,19 @@ in buildPythonPackage rec {
 
   nativeBuildInputs = [
      cmake
+     numpy.blas
```

Comment:
this probably needs to be in both nativeBuildInputs and buildInputs

Reply:
To be honest I don’t really know what packages need to be in buldInputs, nativeBuildInputs or propagatedBuildInputs. This is just something that builds and solves my immediate need. I’d be happy to modify it as needed, but given that each build of pytorch takes a couple of hours I don’t really have the time for a lot of trial and error.